noxfile.py

```diff
@@ -38,6 +38,9 @@ def bench(session: nox.Session) -> None:
 def tests(session: nox.Session) -> None:
     session.install(".[tests]")
     env = {"COVERAGE_FILE": f".coverage.{session.python}"}
+    # Note: Previously used COVERAGE_CORE=sysmon for Python 3.12/3.13 performance,
```

Comment:
remove comment ... please, review everything carefully one more time (PR is very large, you know it better, saves a lot of time if basic cleanup is done)